code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -1689,15 +1689,15 @@ Departamental Swimsuits, for general use
 	species_restricted = null
 
 //Mewchild: Phi Vietsi
-/obj/item/clothing/gloves/fluff/vietsi
+/obj/item/clothing/gloves/ring/seal/fluff/vietsi
 	name = "signet ring"
 	desc = "A signet ring carved from the bones of something long extinct, as a ward against bad luck."
 
 	icon = 'icons/vore/custom_clothes_vr.dmi'
 	icon_state = "vietsi_ring"
 	var/nameset = 0
 
-/obj/item/clothing/gloves/fluff/vietsi/attack_self(mob/user)
+/obj/item/clothing/gloves/ring/seal/fluff/vietsi/attack_self(mob/user)
```

Comment:
If you make it `/obj/item/clothing/gloves/ring/seal/signet/fluff/vietsi`  then you can eliminate this proc entirely as its identical to `/obj/item/clothing/gloves/ring/seal/signet/attack_self`